### PR TITLE
Examples tests: use systemctl instead of the removed snap service

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
Requires #484

File: examples_tests/__init__.py [hunk 1]
```diff
@@ -178,9 +181,7 @@ def run_command_in_snappy_testbed(self, command):
...
         if not config.get('skip-install', False):
             output = self.run_command_in_snappy_testbed(
-                ['sudo', 'snap', 'service', 'status', snap])
-            expected = (
-                'Snap\t+Service\t+State\n'
-                '{}\t+{}\t+enabled; loaded; active \(running\)\n'.format(
-                    snap, service))
-            self.assertThat(output, MatchesRegex(expected))
+                ['systemctl', '--no-pager', 'status',
+                 'snap.{}.{}'.format(snap, service)])
```
Comment (kyrofa) [inline examples_tests/__init__.py]: Huh, this file format must have changed in 2.0, eh?

**Comment by elopio** [comment]
retest this please

File: examples_tests/__init__.py (hunk 1 above)
Comment (elopio) [inline examples_tests/__init__.py]: This is the unit name, and yes, it changed. I don't remember how was it named before; but that shouldn't matter, all of these details should be hidden behind a nice snap command. snap service was one of the commands I liked the most, I hope they bring it back.

Comment (kyrofa) [inline examples_tests/__init__.py]: Yeah, I just hope it gains the ability to access specific services within a snap.

**Comment by kyrofa** [comment]
I'm not a systemd expert, but this looks good to me :+1: .

**Comment by sergiusens** [comment]
@elopio please create a bug with tasks for `snapcraft` and `snappy` so we can be in sync for when it comes back

**Pull request merged**